Answers.md

```diff
@@ -1,5 +1,5 @@
 1.  Explain the differences between `client-side routing` and `server-side routing`.
```

Comment:
I'd like to see you finish answering all of the questions. It will be good practice for interview-type questions.